core/store/models/job_spec.go

```diff
@@ -216,43 +215,22 @@ type InitiatorParams struct {
 	ToBlock    *utils.Big        `json:"toBlock,omitempty" gorm:"type:varchar(255)"`
 	Topics     Topics            `json:"topics,omitempty"`
 
-	RequestData     JSON     `json:"requestData,omitempty" gorm:"type:text"`
-	IdleThreshold   Duration `json:"idleThreshold,omitempty"`
-	Feeds           Feeds    `json:"feeds,omitempty" gorm:"type:text"`
-	Threshold       float32  `json:"threshold,omitempty" gorm:"type:float"`
-	Precision       int32    `json:"precision,omitempty" gorm:"type:smallint"`
-	PollingInterval Duration `json:"pollingInterval,omitempty"`
+	RequestData JSON            `json:"requestData,omitempty" gorm:"type:text"`
+	Feeds       Feeds           `json:"feeds,omitempty" gorm:"type:text"`
+	Precision   int32           `json:"precision,omitempty" gorm:"type:smallint"`
+	Threshold   float32         `json:"threshold,omitempty"`
+	PollTimer   PollTimerConfig `json:"pollTimer,omitempty" gorm:"type:jsonb"`
+	IdleTimer   IdleTimerConfig `json:"idleTimer,omitempty" gorm:"type:jsonb"`
 }
 
-// defaults represents a default value for an initiator parameter. Value should
-// be the default value, and isZero should be a predicate which returns true if
-// the value of an initiator passed to SetDefaultValues should be overwritten
-// with the default value. See FluxMonitorDefaultInitiatorParams.
-type defaults struct {
-	Value  interface{}
-	isZero func(interface{}) bool
+type PollTimerConfig struct {
+	Disabled  bool     `json:"disabled,omitempty"`
+	Frequency Duration `json:"frequency,omitempty"`
```

Comment:
If you change this to `Duration`, you can just have a `TimerConfig`.

Reply:
I think that will cause problems with gorm, it won't know how to differentiate columns for `PollTimer` and `IdleTimer`.